File: Entities/Details/RankingDetail.cs
using System.ComponentModel.DataAnnotations;
using Core.Entities;

namespace Entities.Details
{
    public class RankingDetail : IEntity
    {
        [Key]
        public int RankingDetailId { get; set; }
        public int RankingQuestionId { get; set; }
        public int Rank { get; set; }
        public string Title { get; set; }

    }
}

File: WebUI/UI/src/app/datas/models/details/ranking-detail.ts
export class RankingDetail {
  constructor(rankingQuestionId: number, rank: number, title: string) {
    this.rankingQuestionId = rankingQuestionId;
    this.rank = rank;
    this.title = title;
  }

  rankingDetailId: number;
  rankingQuestionId: number;
  rank: number;
  title: string;
}
